duo-client/src/main/java/com/duosecurity/client/Http.java

```diff
@@ -236,6 +240,10 @@ public void addParam(String name, String value) {
     params.put(name, value);
   }
 
+  public void addAdditionalDuoHeader(Map<String, String> inAdditionalDuoHeaders){
+    additionalDuoHeaders = inAdditionalDuoHeaders;
```

Comment:
you'll need additionalDuoHeaders.putAll(inAdditionalDuoHeaders) here instead
otherwise you'll replace the map you created at line 42 with whatever the user provides, which could be a different type of map; we want to make sure we use our map